google-beta/resource_pubsub_subscription.go

```diff
@@ -63,166 +77,375 @@ func resourcePubsubSubscription() *schema.Resource {
 							Optional: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
-
 						"push_endpoint": {
 							Type:     schema.TypeString,
-							Required: true,
```

Comment:
Just to confirm, this change is intentional?

Reply:
This is a side effect of making the resource updatabl to remove the ForceNew on the parent `push_config` updatable. I couldn't figure out a way to make the parent optional and the child required, should I modify the child with a `required: true` directly on api.yaml or just on terraform.yaml?

Reply:
You should be able to mark the child `required: true` in api.yaml; that's how we indicated a field is required if the nested block is present.